base/collection_n1ql_common.go

```diff
@@ -171,14 +204,12 @@ func createIndex(ctx context.Context, store N1QLStore, indexName string, createS
 
 	if IsIndexerRetryIndexError(err) {
```

Comment:
Could you combine these two if statements to something like `if IsIndexerRetryIndexError(err) || IsCreateDuplicateIndexError(err)`?

I don't think that either of these are necessarily need to be flagged as Info, especially the "duplicate index creation", since I think this is likely to happen in the case that multiple nodes are running through the index creation code simultaneously, in the case of a `PUT /db` with `start_offline: true` on a multi node cluster. The first node will start the initialization and then call `BootstrapConfig.UpdateConfig` for the database, and the subsequent nodes will pick up that config with bootstrap polling. However, with a sufficiently size index creation, the idea that all nodes will exist at the same time is more likely.

Without `start_offline: true`, this would be less likely to happen since the index creation is synchronous to the HTTP rest call.

I think that dropping these statements to debug, or lessening the `Error:` / `error` warning is appropriate. Maybe something like:

```
Index %s already being created on server. N1QL output: %s", indexName, err
```